pkgs/by-name/dp/dprint/plugins/update-plugins.py

```diff
@@ -28,25 +29,17 @@
     pname = os.environ.get("UPDATE_NIX_PNAME", "")
 
 
-# get sri hash for a url, no unpack
-def nix_prefetch_url(url, name, algo="sha256"):
-    hash = (
+# get sha256 hash for a url, no unpack
+def prefetch_url(url: str) -> str:
+    sha256 = (
```

Comment:
instead of the sri hash using the sha256, because we need it to create the `/nix/store/.../plugin.wasm@sha256`

Reply:
not anymore, back to sri hash